ouroboros-consensus-cardano/cddl/node-to-client/localstatequery/query.cddl

```diff
@@ -0,0 +1,105 @@
+query = [0, blockQuery]
+      / [getSystemStart.query]
+      / [getChainBlockNo.query]
+      / [getChainPoint.query]
+
+blockQuery = queryIfCurrent
+           / queryAnyTime
+           / queryHardFork
+
+queryAnyTime = [1, getEraStart.query, eraIdx]
+
+queryHardFork = [2, getInterpreter.query / getCurrentEra.query]
+
+queryIfCurrent =
+  [0, ns7<byronQuery,
+          shelleyQuery,
+          shelleyQuery,
+          shelleyQuery,
+          shelleyQuery,
+          shelleyQuery,
+          shelleyQuery>
+          ]
+
+byronQuery = getUpdateInterfaceState.query
+
+shelleyQuery = getLedgerTip.query
+             / getEpochNo.query
+             / getNonMyopicMemberRewards.query
+             / getCurrentPParams.query
+             / getProposedPParamsUpdates.query
+             / getStakeDistribution.query
+             / getUTxOByAddress.query
+             / getUTxOWhole.query
+             / debugEpochState.query
+             / getCBOR.query
+             / getFilteredDelegationsAndRewardAccounts.query
+             / getGenesisConfig.query
+             / debugNewEpochState.query
+             / debugChainDepState.query
+             / getRewardProvenance.query
+             / getUTxOByTxIn.query
+             / getStakePools.query
+             / getStakePoolParams.query
+             / getRewardInfoPools.query
+             / getPoolState.query
+             / getStakeSnapshots.query
+             / getPoolDistr.query
+             / getStakeDelegDeposits.query
+             / getConstitution.query
+             / getGovState.query
+             / getDRepState.query
+             / getDRepStakeDistr.query
+             / getCommitteeMembersState.query
+             / getFilteredVoteDelegatees.query
+             / getAccountState.query
+             / getSPOStakeDistr.query
+             / getProposals.query
+             / getRatifyState.query
+             / getFuturePParams.query
+             / getBigLedgerPeersSnapshot.query
+
+getCBOR.query = [9, shelleyQuery]
+
+;# include getSystemStart as getSystemStart
```

Comment:
Does this mean that we need to provide `./consensus` as an include path to `cddlc`? Would it be possible to say `consensus/getSystemStart` here and it would just work?

Reply:
Do check the `cddlc.sh` script in this PR, it shows how one would run cddlc. You can run it to see the results.

Reply:
> Would it be possible to say consensus/getSystemStart here and it would just work?

No, I don't think that works. I seem to recall I tried and was not able to use it.